finance-port.patch

```diff
@@ -0,0 +1,102 @@
+diff --git a/finance.py b/finance.py
+index 24022e0..a371d2f 100644
+--- a/finance.py
++++ b/finance.py
+@@ -26,6 +26,7 @@ import json
+ import tempfile
+ import io
+ import dbus
++import shutil
+ 
+ import gi
+ gi.require_version('Gtk', '3.0')
+@@ -42,12 +43,13 @@ from sugar3.graphics import style
+ from sugar3.activity.widgets import StopButton
+ from sugar3.activity.widgets import ActivityToolbarButton
```

Comment:
Replace it for sugarapp's `ExtendedActivityToolbarButton` and it will allow the user to explicitly save and load his data to/from the app.

This will also require to use sugarapp-gen-mimetypes in the manifest, e.g. https://github.com/tchx84/org.sugarlabs.ReadETexts-flathub/blob/master/org.sugarlabs.ReadETexts.json#L301-L305